src/test/java/com/browserstack/automate/AutomateClientTest.java

```diff
@@ -235,6 +235,72 @@ public void testGetSessionVideo() {
     }
   }
 
+  @Test
+  public void testGetSessionConsoleLogs() {
+    // TODO: Verify if logs are non-empty
+    // Cannot currently be tested during in-progress sessions
+    try {
+      String buildId = automateClient.getBuilds().get(0).getId();
+      List<Session> sessions = automateClient.getSessions(buildId);
+
+      String logs = sessions.get(0).getConsoleLogs();
+      assertTrue("Session Console Logs", logs != null);
+
+      logs = automateClient.getSessionConsoleLogs(sessions.get(0).getId());
+      assertTrue("Session Console Logs", logs != null);
+    } catch (BuildNotFound e) {
+      assertTrue(false);
+    } catch (SessionNotFound e) {
+      assertTrue(false);
+    } catch (AutomateException e) {
+      assertTrue(false);
+    }
+  }
+
+  @Test
+  public void testGetSessionHARLogs() {
+    // TODO: Verify if logs are non-empty
+    // Cannot currently be tested during in-progress sessions
+    try {
+      String buildId = automateClient.getBuilds().get(0).getId();
+      List<Session> sessions = automateClient.getSessions(buildId);
+
+      String logs = sessions.get(0).getHARLogs();
+      assertTrue("Session HAR Logs", logs != null);
+
+      logs = automateClient.getSessionHARLogs(sessions.get(0).getId());
+      assertTrue("Session HAR Logs", logs != null);
+    } catch (BuildNotFound e) {
+      assertTrue(false);
+    } catch (SessionNotFound e) {
+      assertTrue(false);
+    } catch (AutomateException e) {
+      assertTrue(false);
+    }
+  }
+
+  @Test
+  public void testGetSessionAppiumLogs() {
```

Comment:
This test will fail in case the last session is run on desktop or appium logs capability was not passed in that.